handlers/meshsync_handlers.go

```diff
@@ -0,0 +1,39 @@
+package handlers
+
+import (
+	"net/http"
+
+	"github.com/layer5io/meshery/models"
+)
+
+// MeshSyncHandler - handles that parses meshsync response
+func (h *Handler) MeshSyncHandler(w http.ResponseWriter, req *http.Request, prefObj *models.Preference, user *models.User, provider models.Provider) {
+
+	data := `{
+		"clusters":[{
+			"name":"",
+			"nodes":[{
+				"name":"",
+				"cpu":"",
+				"ram":"",
+				"status":""
+			}],
+			"namespaces":[{
+				"name":"",
+				"labels":[],
+				"annotations":[]
+			}],
+			"service-meshes":[{
```

Comment:
How are these names determined?

Reply:
This is just a temporary json string. That will be returned for a particular endpoint. @leecalcote , @kumarabd should I remove this?
Finally it will replaced with GraphQL

Reply:
Sounds good. Let's get you / this up and running first and we can iterate on this bit as we go.